.travis.yml

```diff
@@ -1,4 +1,10 @@
 language: objective-c
-xcode_project: VelocidiSDK.xcodeproj # path to your xcodeproj folder
+os: osx
+xcode_workspace: VelocidiSDK.xcworkspace # path to your xcodeproj folder
 xcode_scheme: VelocidiSDKTests
-xcode_destination: platform=iOS Simulator,OS=10.1,name=iPad Pro (9.7-inch)
+xcode_destination: platform=iOS Simulator,OS=11.4,name=iPhone X
```

Comment:
Is it worth to test other iOS versions?

The most recent version is iOS 12 is [installed on 78% of all devices from the last four years (as of January 2019)](https://www.macrumors.com/2019/01/02/ios-12-installed-on-78-percent-of-devices/). The following widest version is iOS 11. Also, while [users on iOS 11 get a chance to update, users on iOS 10 don't](https://everyi.com/by-capability/maximum-supported-ios-version-for-ipod-iphone-ipad.html). The same logic applies to the rest of the iOS versions.

My take is that, depending on the minimum iOS version supported by the SDK, we should test the major versions starting from the minimum. We can take advantage of the [build matrix](https://docs.travis-ci.com/user/build-matrix/) to test the different versions.

What do you think?

Reply:
Tried to do this in f350fdd but was not able to get testing against multiple SDK to work. Still changed the `.travis.yml` to make this easier in the future (just uncomment two lines). Can ignore it for now and come back to it in the future, when we are more comfortable with iOS development? We can open an issue with this

Reply:
Nvm 😄  Added tests for SDK versions:
- 11.0
- 11.1
- 11.2
- 11.3
- 11.4
- 12.0
- 12.1
- 12.2